src/main.rs

```diff
@@ -208,6 +209,47 @@ fn build_nmap_arguments<'a>(
     arguments
 }
 
+/// Goes through all possible IP inputs (files or via argparsing)
+/// Parses the string(s) into IPs
+fn parse_addresses(opts: &Opts) -> Vec<IpAddr> {
+    // Construct a new Resolver with default configuration options, DNS.
```

Comment:
Dead comment, need to delete this. Will delete after the next comment is talked about :)